facet-derive-emit/tests/compile_tests.rs

```diff
@@ -0,0 +1,176 @@
+use std::fs;
```

Comment:
I'd like to extract this to `facet_testhelpers` at some point, instead of having it be duplicated.